### ISSUE TITLE
Ignore certain view rules in refined/extended views

### ISSUE THREAD

**Issue by emorrell23** [spectacles-ci/lkmlstyle#20]
I am finding that some of the View-based rules are having issues with extended and refined views.

* V110 View must define at least one primary key dimension
    * Applies to:
        * Extended views
        * Refined views
    * Issue
        * Both refined views and extended views are being flagged for this rule when the base view should be the only one that is checked.
* V100 
    * Applies to:
        * Refined views
    * Issue
        * Refined views are being flagged because of the “+” at the start of the view name
```
[V100] View name not in snake case
ca_logic/views/pdt.combined_dbt_run_results.view.lkml:3
────────────────────────────────────────────────
 3    >| view: +combined_dbt_run_results {

```

**Comment by joshtemple** [comment]
The remaining part of this is duplicated with #25, so I'm going to close this out in favor of that issue for the remaining work.
